src/Renderer/BaseRenderer.php

```diff
@@ -25,22 +25,22 @@ class BaseRenderer
      * @param string : list of the renderer
      * @param string : base_path
      */
-    public function __construct($renderer, $base_path)
+    public function __construct($renderer, $render_options)
     {
         $rendererList = explode(',', $renderer);
 
         $this->nameList = array();
         $this->rendererList = array();
 
         //let's load the renderer dynamically
-        foreach ($rendererList as $renderer) {
-            $this->nameList[] = $renderer;
-            if (in_array($renderer, array('Behat2', 'Twig', 'Minimal'))) {
-                $className = __NAMESPACE__.'\\'.$renderer.'Renderer';
-            } else {
-                $className = $renderer;
+        foreach ($rendererList as $r) {
```

Comment:
We should probably have a better variable name than r in this loop.  
```suggestion
        foreach ($rendererList as $renderer) {
```